docs/guide/server-side-rendering.md

```diff
@@ -50,6 +50,8 @@ export default defineNuxtConfig({
 })
 ```
 
+For a more detailed example, see the [SASS Modern Compiler doc](https://github.com/vuetifyjs/nuxt-module/blob/main/docs/guide/sass-modern-compiler.md#overriding-sass-variables).
```

Comment:
replace the link removing the url: `[SASS Variables](/guide/sass-modern-compiler.md#overriding-sass-variables)`.